### DISCUSSION TITLE
AR doesn't start with Panning listeners in react

### DISCUSSION THREAD

**Discussion by zavalit** [google/model-viewer#2859] [Q&A]
If i try to start AR mode on my mobile device nothing  happens. But if i put an `alert` on place where it skips a `touchstart` event (lines 80, 81 in gist example), it starts to launch an AR Scene.

#### Live Demo
https://gist.github.com/zavalit/383edccdd129274f1e96a7f452ef44e2#file-panningmodel-jsx-L80-L81

### Version
- model-viewer: v1.8.0
- react: v17.0.2

### Browser Affected
- [x] Chrome, version: 94.0.4606.61

### OS
- [x] Android
- [x] iOS

### AR
- [x] WebXR
- [x] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
Can you repro this behavior on a plain HTML page, perhaps by remixing our Glitch? https://glitch.com/edit/#!/model-viewer

If not, then it sounds like a React bug.
